django/template/loaders/cached.py

```diff
@@ -17,6 +17,13 @@
 
 class Loader(BaseLoader):
 
+    def get_dirs(self):
+        dirs = set()
+        for loader in self.loaders:
+            if loader is not None:
+                dirs.update(loader.get_dirs())
+        return dirs
+
```

Comment:
I thought I will eventually get to that kind of answer on the PR. Was thinking about some similar solution, but wasn't able to write it, then I pushed something that I knew was working.